stable/prometheus-rabbitmq-exporter/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for prometheus-rabbitmq-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: kbudde/rabbitmq-exporter
+  tag: v0.28.0
+  pullPolicy: IfNotPresent
+service:
+  type: ClusterIP
+  externalPort: 9419
+  internalPort: 9419
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+loglevel: info
+rabbitmq:
+  url: http://myrabbit:15672
+  user: guest
+  password: guest
+  capabilities: bert,no_sort
+  include_queues: ".*"
+  skip_queues: "^$"
+
+annotation:
```

Comment:
Pleae comment the annotations out. They should be optional. You don't need them if you use prometheus-operator.